client/landing/stepper/declarative-flow/internals/steps-repository/index.ts

```diff
@@ -19,6 +20,12 @@ export type StepPath =
 	| 'bloggerStartingPoint'
 	| 'storeFeatures'
 	| 'designSetup'
+	| 'import'
+	| 'import/list'
+	| 'import/ready'
+	| 'import/ready/not'
+	| 'import/ready/wpcom'
+	| 'import/ready/preview'
```

Comment:
Unless it's a deal breaker for you, I'd aim to maintain the flatness of the steps. As in change `import/list` to `import-list`, `import-ready`, etc.. 

I entirely agree these steps are part of `import`, but it would be nice to establish a sense of flat steps-repository; every step is simple a nice component connected to the store. This makes every step directly reusable in other flows when we inevitably create them.

```suggestion
	| 'import-list'
	| 'import-ready'
	| 'import-ready-not'
	| 'import-ready-wpcom'
	| 'import-ready-preview'
```

Then you'll be free from navigation magic as well, you won't need to remove trailing slashes.

Reply:
My idea was to reuse the same route structure as we have on `/start/import/{STEP}/{SECTION}`, but it's not a deal-breaker to update routes as you proposed (it's always good to be consistent).

Reply:
It has been updated. I've changed slashes routes to the camel case pattern to follow existing route patterns.

`import/ready/preview` -> `importReadyPreview`